2025/build-tooling/vite.config.ts

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+  server: {
+    fs: {
+      allow: ['../..'],
```

Comment:
to reduce duplication, dependencies are installed at the monorepo root, rather than inside the package.
need to add this server.fs.allow setting to allow vite to read files from outside the package